app/models/outgoings_payment.rb

```diff
@@ -4,7 +4,13 @@ class OutgoingsPayment < ApplicationRecord
   attribute :amount, :pence
   attribute :payment_type, :value_object, source: HousingPaymentType
 
-  store_accessor :metadata, :details, :case_reference
+  store_accessor :metadata,
+                 :details,
+                 :case_reference,
+                 :board_amount,
+                 :food_amount,
+                 :payee_name,
+                 :payee_relationship_to_client
```

Comment:
@RoseSAK note I've made board_amount etc all dedicated virtual fields so that are strongly typed in Schema and also allows the forcing of `board_amount` and `food_amount` to Integers, again conforming to Schema.